core/compatibility/components/blocks/MyAccount/MyNewsletter.js

```diff
@@ -1,35 +1,7 @@
+import { Newsletter } from '@vue-storefront/core/modules/newsletter/components/Newsletter'
+
```

Comment:
i guess we can get rid of this file and use module directly

Reply:
OK will have a look at it tomorrow!

Reply:
I have removed the file and imported from the module.